modules/claims_api/app/controllers/concerns/claims_api/v2/power_of_attorney_validation.rb

```diff
@@ -40,6 +40,32 @@ def build_dependent_claimant_verification_service(veteran_participant_id:, user_
                                                             poa_code:)
       end
 
+      # the Claimant object is already being validatated below,
+      # so that code has been adjusted and it is being left out of this workflow
+      def validate_address_values
```

Comment:
Could we rename the method to capture the comment? Something like:

```suggestion
      def validate_non_claimant_address_values
```